docs/_static/js/custom.js

```diff
@@ -0,0 +1,7 @@
+
+
+/** Opens links to external docs in a new tab and links to docs internal to the repo in the current tab */
```

Comment:
Maybe add a note and link to the stackoverflow answer this was pulled from.

Reply:
See commit ef92712